src/utils/builder.ts

```diff
@@ -66,6 +67,9 @@ export function exportPathway(
   if (cpg) {
     const exporter = new CPGExporter(pathwayWithElm, pathways, criteria);
     pathwayToExport = exporter.export();
+  } else {
+    const exporter = new CaminoExporter(pathwayWithElm, criteria);
```

Comment:
Do we want the CaminoExporter to consider pathway reference nodes? I don't think that should be part of this task but perhaps a back log item to download all of them in a zip file

Reply:
Yeah let's make that a separate task. I was talking to Reece about that on slack:
> The pathway -> cpg translation creates new IDs at export time, so if you exported A then B to CPG separately, any reference from A to B would be lost, so we have to export them together. Whereas with the internal data model it only links by name, so if you export A then B, the relationship is preserved.

Reply:
Created [PATHWAYS-381](https://jira.mitre.org/browse/PATHWAYS-381)